certified-connectors/Delinea Secret Server/apiDefinition.swagger.json

```diff
@@ -0,0 +1,741 @@
+          "type": "string"
+        },
+        "generatePasswordLength": {
+          "description": "Length of value to generate for this field.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "hideOnView": {
+          "description": "Whether to display this field in a view context.",
+          "type": "boolean"
+        },
+        "historyLength": {
+          "description": "Length of the history stored for this field.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "isExpirationField": {
+          "description": "Whether Secret Expiration is based on this field.",
+          "type": "boolean"
+        },
+        "isFile": {
+          "description": "Whether this field represents a File Attachment.",
+          "type": "boolean"
+        },
+        "isIndexable": {
+          "description": "Whether this field will be indexed for search.",
+          "type": "boolean"
+        },
+        "isList": {
+          "description": "Whether this field is a List field.",
+          "type": "boolean"
+        },
+        "isNotes": {
+          "description": "Whether this field is a Notes field.",
+          "type": "boolean"
+        },
+        "isPassword": {
+          "description": "Whether this field is a Password field.",
+          "type": "boolean"
+        },
+        "isRequired": {
+          "description": "Whether population of this field is required for the secret to be valid.",
+          "type": "boolean"
+        },
+        "isUrl": {
+          "description": "Whether this is a field is a URL field.",
+          "type": "boolean"
+        },
+        "listType": {
+          "$ref": "#/definitions/ListType"
+        },
+        "mustEncrypt": {
+          "description": "Whether this field is encrypted within the Database.",
+          "type": "boolean"
+        },
+        "name": {
+          "description": "Name",
+          "type": "string"
+        },
+        "passwordRequirementId": {
+          "description": "Id of the Password Requirement associated with this field.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "passwordTypeFieldId": {
+          "description": "Id of the Password Type for Web Password Automatic Password Changing.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "secretTemplateFieldId": {
+          "description": "Id of the Field definition on which this Field is based.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "sortOrder": {
+          "description": "Order in which this field is displayed when viewing the Secret.",
+          "type": "integer",
+          "format": "int32"
+        }
+      },
+      "type": "object"
+    },
+    "EditRequiresOptions": {
+      "description": "Reasons that the user may not be able to retrieve the secret.",
+      "properties": {},
```

Comment:
@tylerezimmerman you can remove empty properties if they're empty. They [can be removed unless required](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-exact-operation-responses).